test/is.test.js

```diff
@@ -90,6 +90,25 @@ describe('is', () => {
     });
   });
 
+  describe('asyncFunction', () => {
+    it('should true', () => {
+      var asyncFun = async function() {
```

Comment:
test will failed in node < 7.6, so you should create a standalone test file, detect node version and require it.